docs/csiaddonsjob.md

```diff
@@ -0,0 +1,44 @@
+# CSIAddonsJob
+
+CSIAddonsJob is a namespaced custom resource designed to invoke operation to be performed on a volume.
+
+```yaml
+apiVersion: csiaddons.openshift.io/v1beta1
+kind: CSIAddonsJob
+metadata:
+  name: sample-1
+spec:
+  operation:
```

Comment:
dont we need a schedule here? so that every interval job will be triggered?

Reply:
> dont we need a schedule here? so that every interval job will be triggered?

`csiAddonsJob` will run only once, whereas `csiAddonsCronJob` will run at regular interval which will be added soon.